homeassistant/components/switch/pulseaudio_loopback.py

```diff
@@ -9,69 +9,75 @@
 import socket
 from datetime import timedelta
 
+import voluptuous as vol
+
 import homeassistant.util as util
-from homeassistant.components.switch import SwitchDevice
-from homeassistant.util import convert
+from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
+from homeassistant.const import (CONF_NAME, CONF_HOST, CONF_PORT)
+import homeassistant.helpers.config_validation as cv
 
 _LOGGER = logging.getLogger(__name__)
 _PULSEAUDIO_SERVERS = {}
 
-DEFAULT_NAME = "paloopback"
-DEFAULT_HOST = "localhost"
-DEFAULT_PORT = 4712
+CONF_BUFFER_SIZE = 1024
```

Comment:
DEFAULT_BUFFER_SIZE I think

On Sep 5, 2016 1:10 PM, "Dan Cinnamon" <notifications@github.com> wrote:

> In homeassistant/components/switch/pulseaudio_loopback.py
> <https://github.com/home-assistant/home-assistant/pull/3160#discussion_r77551321>
> :
>
> >
> >  _LOGGER = logging.getLogger(__name__)
> >  _PULSEAUDIO_SERVERS = {}
> >
> > -DEFAULT_NAME = "paloopback"
> > -DEFAULT_HOST = "localhost"
> > -DEFAULT_PORT = 4712
> > +CONF_BUFFER_SIZE = 1024
>
> I think you meant for this to be CONF_BUFFER_SIZE = "buffer_size"
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3160/files/31f2a01e4a0bb7f93f2144fcd4a773f703e1dcda#r77551321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZW2yPuZN1mB4rYyf4aBTaQfx9-2vks5qnGk3gaJpZM4J0VG1>
> .
>